.gitignore

```diff
@@ -16,3 +16,4 @@ out/
 
 
 /src/test/java
+/src/test/resources/schema.sql
```

Comment:
WDYT about adding `/src/test/` here instead of two lines?

Reply:
Good point - done